src/api/db/data/20180214132015_project_log_entry_user_name.rb

```diff
@@ -0,0 +1,14 @@
+class ProjectLogEntryUserName < ActiveRecord::Migration[5.1]
+  def up
+    ProjectLogEntry.where('event_type like "%comment_for%"').each do |log_entry|
```

Comment:
Did you try this with a db dump? We have around ~12k entries for this query and 3M for ProjectLogEntry so I would be curious how long this migration takes.

Reply:
I did not no.

Reply:
Do you plan to do it?

Reply:
Nope why?